selene_sdk/targets/genomic_features.py

```diff
@@ -125,6 +125,8 @@ def _get_feature_data(chrom, start, end,
     get_feature_rows : types.FunctionType
         A function that takes coordinates and returns rows
         (`list(tuple(int, int, str))`).
+    strand : {'+', '-', '.'}, optional
+            Default is '+'. The strand the sequence is located on. '.' is treated
```

Comment:
what does `'.' is treated` mean?

Reply:
My bad - I wanted to be consistent with the rest of the code but this part got cut off :P it's fixed now.